community-connectors/openweathermap-current/main.js

```diff
@@ -0,0 +1,221 @@
+function getConfig(request) {
+
+  var cc = DataStudioApp.createCommunityConnector();
+  var config = cc.getConfig();
+
+  config
+    .newTextInput()
+    .setId('apiKey')
```

Comment:
Use the `KEY` auth type instead of doing credentials through the config. See [Authentication] for full details on what to do.

[Authentication]: https://developers.google.com/datastudio/connector/auth#getauthtype

Reply:
Thanks. I've updated to using the KEY authentication. Let me know if this looks okay.